### DISCUSSION TITLE
No resolution option in iOS?

### DISCUSSION THREAD

**Discussion by Alsen99** [ppy/osu#22555] [Q&A] [answered]
I was trying to play osu with my iPad 6th gen but the max fps it can get is around 20fps, and then I got an idea to just lower the resolution so I go through the settings but it's not there?

I even went to PC to check if the option was there and yes there is, which means you can't change resolution in iOS? Or is it mobile in general?

**Comment by mk56-spn** [comment, marked as answer]
Please wait for further optimisations for mobile before requesting this type of feature. The intent currently is to mainly solve performance through the use of Veldrid ( and the backends it supports ), which has a PR open for implementation currently on the OSU! Framework repo, I suggest waiting until that is online atleast before passing judgement on performance

**Reply by Alsen99** [reply]
Well okay, thank you!
